### ISSUE TITLE
How to link to a local c library using the full path

### ISSUE THREAD

**Issue by BlackEdder** [E-xyza/zigler#435]
I am trying to link to a locally build library based on: https://github.com/E-xyza/zigler/blob/9d5ea0081fe13edfe63471923b2f9495691fc13b/lib/zig.ex#L202
but get the following issue

```bash
Running tests...
/tmp/Elixir.TermBox2Ex/build.zig:34:23: error: expected type 'Build.LazyPath', found '*const [81:0]u8'
    lib.addObjectFile("/home/edwin/Documents/code/rttt-project/rtttex/apps/termbox2_ex/src/libtermbox2.a");
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/edwin/Documents/code/rttt-project/rtttex/deps/zigler/zig/zig-linux-x86_64-0.11.0/lib/std/Build.zig:1672:22: note: union declared here
pub const LazyPath = union(enum) {
                     ^~~~~
/home/edwin/Documents/code/rttt-project/rtttex/deps/zigler/zig/zig-linux-x86_64-0.11.0/lib/std/Build/Step/Compile.zig:1013:46: note: parameter type declared here
pub fn addObjectFile(self: *Compile, source: LazyPath) void {
```

**Comment by ityonemo** [comment]
ah, ok, this is a bug in zigler creating build.zig =/ I will try to get to this this week.

**Comment by ityonemo** [comment]
ok, I got really sick with a cold, so I was unable to get to it, Apologies, but i'm going to punt this to zigler 0.12 because the build system is fluctuating quite a bit.

**Comment by ityonemo** [comment]
ok zig 0.11.1 looks imminent, so I'm going to roll a real quick 0.11.1 release.

**Comment by ityonemo** [comment]
fixed in 0.11.1-pre and will release with 0.11.1 once zig 0.11.1 is finalized.
